.changeset/rare-singers-refuse.md

```diff
@@ -0,0 +1,5 @@
+---
+'xstate-viz-app': patch
+---
+
+Updated XState to its latest version ([4.26.0](https://github.com/statelyai/xstate/releases/tag/xstate%404.26.0)). Visualizer should be able to use the new goodies now, such as it should provide access to the `invoke.meta` object in the invoke creators.
```

Comment:
Maybe we should remove the "such as" note? I'm not sure - I've kinda only included that because it prompted me to do this update but the release itself contains more than that. WDYT?

Reply:
It's fine as-is 👍